projects/igniteui-angular/src/lib/core/styles/components/expansion-panel/_expansion-panel-theme.scss

```diff
@@ -114,6 +114,10 @@
         overflow: hidden;
     }
 
+    %igx-expansion-panel--expanded {
```

Comment:
Does this apply to.. every expansion panel even outside of an accordion? Ignore if this is just temporary

Reply:
Тhe igx-accordion-component set this class only to its expanded children(igxExpansionPanels). However if our customers decide to set manually this class to an igxExpansionPanel outside of an igxAccordion - it will be applied as well.